sr_description/mujoco_models/README.md

```diff
@@ -0,0 +1,24 @@
+# Hand E Mujoco Models
+
+This directory contains files necessary for a Mujoco simulation of the dexterous hand (Hand E).
+
+## Under-Actuation Logic
+
+The simulation correctly models the under-actuation of J1 and J2 of the first, middle, ring and little fingers by defining one tendon for each J1-J2 pair. The model defines two seperate joints, but only one actuator for the pair (J0), which drives the tendon. The tendon is abstract, with it's "length" being the combined angles of J1 and J2. [mujoco_ros_control](https://github.com/shadow-robot/mujoco_ros_pkgs/tree/kinetic-devel/mujoco_ros_control) has been modified to report joint angles from the two seperate joints, but half of the tendon effort for each joint's effort. Additionally, effort or positions commands received for J1 and J2 are summed and applied to J0. This is a reasonable approximation of the real Hand E, with the exception that the real Hand E tends to actuate J2 before J1 due to internal friction.
+
+## Explanation of Files
+
+* [rh_trajectory_controller.yaml](rh_trajectory_controller.yaml)
+    * Contains config for the joint trajectory controller. PID values were based on those used in the pre-existing Gazebo simulation, with anti-windup added when integral windup was observed to be an issue.
+* [shared_assets.xml](shared_assets.xml)
+    * Contains assets (e.g. mesh and material definitions) likely to be common to dexterous hand models (Hand E, G etc.).
+* [shared_options.xml](shared_options.xml)
+    * Contains simulation options likely to be common to dexterous hand models (Hand E, G etc.). Seperate to shared_assets.xml due to Mujoco inclusion syntax (see sr_hand_e_environment.xml).
```

Comment:
"Separate to"?

Reply:
Fixed in b37bfc5150bdb66f16f84b50d3510091dc37444f

Reply:
Actually fixed in 07812ab54bd43791f4d5e75d201fe38c5c717ee4